Zai-glm-best-practices-guide.html

```
@@ -0,0 +1,1799 @@
+          </tr>
+          <tr>
+            <td>GLM-OCR</td>
+            <td>文書レイアウト解析・OCR特化</td>
+            <td>-</td>
+            <td>-</td>
+          </tr>
+        </table>
+
+        <div class="diagram-wrap">
+          <div class="mermaid">
+            flowchart TD A["Z.AI Open Platformにアクセス"] --> B["アカウント登録・ログイン"] B -->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)